commercetools-models/src/test/java/io/sphere/sdk/orders/LineImportDraftBuilderIntegrationTest.java

```diff
@@ -0,0 +1,60 @@
+package io.sphere.sdk.orders;
+
+import io.sphere.sdk.carts.ItemState;
+import io.sphere.sdk.channels.Channel;
+import io.sphere.sdk.models.LocalizedString;
+import io.sphere.sdk.models.Reference;
+import io.sphere.sdk.products.Price;
+import io.sphere.sdk.products.attributes.AttributeImportDraft;
+import io.sphere.sdk.states.State;
+import io.sphere.sdk.taxcategories.TaxRate;
+import io.sphere.sdk.test.IntegrationTest;
+import io.sphere.sdk.types.CustomFieldsDraft;
+import io.sphere.sdk.types.CustomFieldsDraftBuilder;
+import org.junit.Test;
+
+import java.util.Set;
+
+import static io.sphere.sdk.taxcategories.TaxCategoryFixtures.withUpdateableTaxCategory;
+import static io.sphere.sdk.test.SphereTestUtils.*;
+import static io.sphere.sdk.utils.SphereInternalUtils.asSet;
+import static java.awt.Color.yellow;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LineImportDraftBuilderIntegrationTest extends IntegrationTest {
```

Comment:
Please remove `extends IntegrationTest`, since this test doesn't require a running backend service and is just an unit test.